### DISCUSSION TITLE
Finite Elasticity ComputeElasticityTensor is not a registered object

### DISCUSSION THREAD

**Discussion by RbmIjpb** [idaholab/moose#23030] [Q&A Modules: Solid mechanics] [answered]
Hello, 
I'm trying to run the input file from tutorials (modules/tensor_mechanics/test/tests/finite_strain_elastic/finite_strain_elastic_new_test.i) 
and I get this error. Of course the TENSOR MECHANICS was activated in the Makefile.

*** ERROR ***
A 'ComputeElasticityTensor' is not a registered object.

If you are trying to find this object in a dynamically linked library, make sure that
the library can be found either in your "Problem/library_path" parameter or in the
MOOSE_LIBRARY_PATH environment variable.

Can someone please help me.
Thanks

**Comment by GiudGiud** [comment, marked as answer]
Hello

Is this the makefile from an application you are mentioning? How did you activate it?
Did you rebuild after selecting it?
Does it work as expected with the tensor mechanics module executable?

Guillaume
